.github/workflows/release.yaml

```diff
@@ -15,11 +15,11 @@ jobs:
       contents: read
       id-token: write
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
         with:
-          node-version: '18.x'
-          registry-url: 'https://registry.npmjs.org'
+          node-version: "20.x"
```

Comment:
curious if we need to update our require node version docs anywhere else ? not sure.

Reply:
Hmmm. I moved to `20.x` because that's the version specified in `package.json`. I'll update the README here as well. If we see anywhere else that's on 18.x, let's update it there too.